cerberus/kubernetes/client.py

```diff
@@ -80,6 +82,41 @@ def check_sdn_namespace():
         please specify the correct networking namespace in config file")
 
 
+def namespace_sleep_track(namespace):
+    global pods_tracker
+    pods = list_pods(namespace)
+    crashed_restarted_pods = []
+    for pod in pods:
+        try:
+            pod_info = cli.read_namespaced_pod_status(pod, namespace,
+                                                      pretty=True)
+        except ApiException as e:
+            logging.error("Exception when calling \
+                           CoreV1Api->read_namespaced_pod_status: %s\n" % e)
+        pod_status = pod_info.status
+        pod_status_phase = pod_status.phase
+        pod_restart_count = 0
+        if pod_status_phase != "Succeeded":
+            pod_creation_timestamp = pod_info.metadata.creation_timestamp
+            if pod_status.container_statuses:
+                for container in pod_status.container_statuses:
+                    pod_restart_count += container.restart_count
+            if pod_status.init_container_statuses:
+                for container in pod_status.init_container_statuses:
+                    pod_restart_count += container.restart_count
+            if pods_tracker[pod]:
+                if pods_tracker[pod]["creation_timestamp"] != pod_creation_timestamp or \
+                    pods_tracker[pod]["restart_count"] != pod_restart_count:
+                    crashed_restarted_pods.append(pod)
+                    pods_tracker[pod]["creation_timestamp"] = pod_creation_timestamp
+                    pods_tracker[pod]["restart_count"] = pod_restart_count
+            else:
+                crashed_restarted_pods.append(pod)
```

Comment:
The list of crashed/restarted pods is currently the pod name.  A possible improvement is to append <pod_namspace>:<pod_name> to the list to make it clear where the pod is.   This is minor though, I am fine if we move forward like it is now.